platform/darwin/src/MGLAccountManager.m

```diff
@@ -42,7 +45,14 @@ + (void)load {
     }
 
 #if TARGET_OS_IPHONE || TARGET_OS_SIMULATOR
-    self.skuToken = [MBXSKUToken tokenForSKUID:MBXAccountsSKUIDMaps type:MBXAccountsSKUTypeUser];
+    // Check if we should use an external accounts library (e.g., provided by navigation)
+    id externalAccountsClass = NSClassFromString(MGLAccountManagerExternalClassName);
+    SEL externalSKUTokenMethod = NSSelectorFromString(MGLAccountManagerExternalMethodName);
+    if (externalAccountsClass != nil && [externalAccountsClass respondsToSelector:externalSKUTokenMethod]) {
```

Comment:
This creates an expectation that `externalSKUTokenMethod` will always behave the same way as `MBXSKUToken.skuToken`. One way to make this weak linking mechanism more robust would be to conditionalize on the accounts framework’s `mbxaccountsVersion` constant or some other numeric constant that we can bump whenever something significant changes about this framework. But this is probably robust enough for a temporary solution.

Reply:
I really, really hope this ends up being a temporary solution.